src/aura/CRLP_CreateRollup/CRLP_CreateRollupController.js

```diff
@@ -0,0 +1,37 @@
+({
+    doInit: function(cmp, event, helper){
+        var labels = cmp.get("v.labels");
+        var templateList = [{label: labels.noRollupType, summaryObject: labels.na, detailObject: labels.na}
+            , {label: labels.opportunityLabel + ' -> ' + labels.accountLabel + ' ' + labels.hardCredit,
+            summaryObject: 'Account', detailObject: 'Opportunity'}
+            , {label: labels.opportunityLabel + ' -> ' + labels.accountLabel + ' ' + labels.softCredit
+                , summaryObject: 'Account', detailObject: 'npe01__OppPayment__c'}
+            , {label: labels.opportunityLabel + ' -> ' + labels.contactLabel + ' ' + labels.hardCredit
+                , summaryObject: 'Account', detailObject: 'Partial_Soft_Credit__c'}
```

Comment:
Definitely, and that's partly why we're retrieving all of our labels in Apex with the format of SObjectType.Partial_Soft_Credit__c.getDescribe().getName() you mentioned. The npsp__Rollup__mdt would be worth discussing though because I don't think that's what we're expecting right now.

Reply:
All of the objects and fields included in the Cumulus package will be namespaced with "npsp". That includes the Allocation__c, Partial_Soft_Credit__c, General_Accounting_Unit__c, Error__c, Rollup__mdt, Filter_Group__mdt, etc. objects as well as all fields on those objects and any fields added to objects in the other NPSP packages (such as npo02__Households_Settings__c.npsp__Use_Dated_Conversion_Rates__c).